lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -506,6 +508,18 @@ def initialize_inbound_handlers
     @inbound_handlers = []
   end
 
+  #
+  # Decrypt the given packet with the appropriate key depending on
+  # if this session is a pivot session or not.
+  #
+  def decrypt_inbound_packet(packet)
+    pivot_session = self.find_pivot_session(packet.session_guid)
+    tlv_enc_key = self.tlv_enc_key
+    tlv_enc_key = pivot_session.pivoted_session.tlv_enc_key if pivot_session
+    packet.from_r(tlv_enc_key)
+    packet
+  end
+
```

Comment:
Pretty sure this breaks `getsystem`.

Reply:
![tenor](https://user-images.githubusercontent.com/28896/80688126-aeff6e80-8b0e-11ea-9cfd-6147fffa46bc.gif)